docs/3.0/resources/upgrade-to-prefect-3.mdx

```diff
@@ -1,19 +1,28 @@
 ---
-title: Upgrade to Prefect 3
-description: Learn how to upgrade from Prefect 2 to Prefect 3.
+title: Upgrade to Prefect 3.0
+description: Learn how to upgrade from Prefect 2.x to Prefect 3.0.
 ---
 
-Prefect 3 introduces exciting new features and improvements while maintaining compatibility with most Prefect 2 workflows. For the majority of users, upgrading to Prefect 3 will be a seamless process that requires few or no code changes. This guide highlights key changes that you may need to consider when upgrading.
+Prefect 3.0 is now the default open source version!
```

Comment:
This sentence is odd to me - I don't think 3.0 needs an introduction on this page